<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="LP-5CG0471MNB" failures="1" tests="5" name="test2" time="0.125" errors="0" timestamp="2024-03-16T17:27:01 IST">
  <testcase classname="Test1.day1" name="Demo" time="0.029">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at Test1.day1.Demo(day1.java:30)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:833)
... Removed 21 stack frames]]>
    </failure>
  </testcase> <!-- Demo -->
  <testcase classname="Test1.day1" name="Dinging" time="0.002"/>
  <testcase classname="Test1.day1" name="apiTestCarLoan" time="0.003"/>
  <testcase classname="Test1.day1" name="mobileTestCarLoan" time="0.003"/>
  <testcase classname="Test1.day1" name="webTestCarLoan" time="0.003"/>
</testsuite> <!-- test2 -->
